src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -433,6 +446,20 @@ public CaseDetails createCaseEventForCitizen(
         return searchOperation.execute(metadata, sanitizedParams);
     }
 
+    @RequestMapping(value = "/searchCases", method = RequestMethod.POST)
+    @ApiOperation("Search case data according to the given ElasticSearch query")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "List of case data for the given search query")
+    })
+    public CaseDetailsSearchResult searchCases(
+            @ApiParam(value = "Case type ID", required = true)
+            @RequestParam("ctid") List<String> caseTypesId,
```

Comment:
Typo? caseTypesId or caseTypeIds? So this new API will support search across multiple case type.

Reply:
correct will support search across multiple types. `caseTypesId` is correct in this case right? (sometimes my English fails me :) )

Reply:
Well caseTypeIds sounds more logical as it is a list of ids :-). Its a minor one though, you can ignore.

Reply:
done